CONTRIBUTE.md

```diff
@@ -0,0 +1,31 @@
+## How to contribute to Next.js for Acquia CMS
+
+> **_NOTE:_** You can use the web server of your choice.
+
+1. Have Acquia CMS installed from https://github.com/acquia/next-acms/blob/main/README.md#installation-acquia-cms with your Node environment variables copied to your clipboard.
+
+![Screenshot of environment variables example on ACMS](assets/env_variables.png)
+
+3. Fork the `acquia/next-acms` project repo and pull it down to your local machine (outside of the ACMS directory is recommended) and add the original repo as the upstream.
+
+```
+$ cd next-acms
+$ git remote add upstream https://github.com/acquia/next-acms
+```
+
+4. Create an `.env.local` file in the `/starters` directory in your forked repo and paste in your environment variables from step 1.
+5. From the root directory, run `yarn install,` `yarn build`, then `yarn dev` to start the development server on `http://localhost:3000`.
+
+   > **_NOTE:_** Next.js preview mode only works in production mode.
```

Comment:
Can we mention that this means running `yarn build` and then running `yarn start` inside the `starters/basic-starter` directory